assets/messages.yml

```diff
@@ -257,25 +250,32 @@ messages:
 
         %quick_links%
       fillname: []
-  auth-server-down:
-    - project: mc
-      name: Auth server down
-      shortcut: auth
-      category: invalid
+    - project:
+        - mcpe
```

Comment:
Do we want to have BDS listed in the here as well? On line 240-241

Reply:
Done at https://github.com/mojira/helper-messages/pull/249/commits/932d43591fb60b4608f014a26c42508799bd31a0